dev-lisp/ecls/files/README.gentoo

```diff
@@ -0,0 +1,15 @@
+-*- outline -*-
+
+Gentoo GNU/Linux specific notes for ECL
+---------------------------------------
+
+This is the README.Gentoo file from /usr/share/doc/@PF@/ directory.
+
```

Comment:
I wonder about this line. I guess the idea once was to replace `@PF@` by the actual version, but existing ebuilds don't seem to do that.

Maybe remove the line entirely, it doesn't add much useful information.

Reply:
This line really says "you are here". I'll remove it.